package.json

```diff
@@ -154,6 +154,7 @@
     "lazypipe": "1.0.2",
     "list-imports-exports": "0.1.2",
     "lolex": "5.1.2",
+    "magic-string": "0.25.7",
```

Comment:
@jridgewell would you mind explaining the need for magic string again, vs just remapping? is it just purely to splice the 3p library code into the source?